.github/workflows/scorecard.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           results_format: sarif
           publish_results: true
       - name: Upload artifact
-        uses: actions/upload-artifact@97a0fba1372883ab732affbe8f94b823f91727db # v3.pre.node20
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.pre.node20
```

Comment:
@mmorel-35 only the hash is updated here. How do we update the comment?

Reply:
Labels are being updated in newer PRs, closing this for now, renovate will update in future.